set.mm

```diff
@@ -557814,6 +557824,22 @@ dependency is expressed in our hypothesis (called implicit
     $( $j usage 'wl-cbvalsbi' avoids 'ax-11'; $)
   $}
 
+  ${
+    $d x y w $.  $d ph w $.  $d ps w $.  $d w z $.
+    wl-sbievg.1 $e |- F/ y ph $.
+    wl-sbievg.2 $e |- F/ x ps $.
+    wl-sbievg.3 $e |- ( x = y -> ( ph <-> ps ) ) $.
+    $( Substitution applied to expressions linked by implicit substitution.
+       Credits for the proof go to Gino Giotto, who devised it as part of a
+       former ~ cbvabw version, as of 23-May-2024.  (Contributed by Wolf
+       Lammen, 26-Oct-2024.) $)
+    wl-sbievg $p |- ( [ z / x ] ph <-> [ z / y ] ps ) $=
+      ( vw weq wi wal wsb nfv nfim equequ1 imbi12d cbvalv1 imbi2i df-sb 3bitr4i
+      albii ) IEJZCIJZAKZCLZKZILUCDIJZBKZDLZKZILACEMBDEMUGUKIUFUJUCUEUICDUDADUD
+      DNFOUHBCUHCNGOCDJUDUHABCDIPHQRSUBACIETBDIETUA $.
+    $( $j usage 'wl-sbievg' avoids 'ax-10' 'ax-13'; $)
+  $}
```

Comment:
I think this theorem should be moved to the main part. The comment could be "Substitution applied to expressions linked by implicit substitution.  Part of a former ~ cbvabw version, (Contributed by Gino Giotto,23-May-2024.)  (Revised by Wolf Lammen, 26-Oct-2024.)"

Reply:
I'd rather go with (Contributed by GG and WL, 26-Oct-2024.)